frontend/event/eventController.js

```diff
@@ -176,15 +173,16 @@
             }
         }
 
+        /*
+        * This function remove the string '.000Z' added in end of properties start_time and end_time automatically by generatePatch.
+        * @param {patch} - The patch list of properties that be changed.
```

Comment:
"[...] that was changed."

You have to use the correct form of the aux verb to use in a sentence. 
**Read suggestion**: http://www.talkenglish.com/grammar/auxiliary-verbs-be-do-have.aspx